components/shooter.py

```diff
@@ -135,3 +146,4 @@ def execute(self) -> None:
 
         flywheel_request = VelocityVoltage(self.desired_flywheel_speed)
         self.flywheel.set_control(flywheel_request)
+        self.flywheel1 = Follower(TalonIds.shooter_flywheel, True)
```

Comment:
Looking at the examples on [https://v6.docs.ctr-electronics.com/en/latest/docs/migration/migration-guide/control-requests-guide.html](url)
It looks like you can set flywheel1 control request with Follower but can’t assign Follower with TalonFX.
`self.flywheel1.set_control(Follower(…))`